manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://download.ykt.cbern.com.cn/pc_ver/%E6%99%BA%E6%85%A7%E4%B8%AD%E5%B0%8F%E5%AD%A6%20Setup%201.0.0.exe
+  InstallerSha256: EDC09E439189E11D95C7BFD6265B7A5DF5AE6D2E152B6D5970F65B49254C54AF
```

```suggestion
  InstallerSha256: EDC09E439189E11D95C7BFD6265B7A5DF5AE6D2E152B6D5970F65B49254C54AF
  ProductCode: 'c6cc9519-7239-555e-b960-aa99cc131aac'
```
The product code will help identifying when the package is installed